examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -380,9 +380,9 @@ func (n *dcFnNamer) Name(t *types.Type) string {
 	pubName := n.public.Name(t)
 	n.tracker.AddType(t)
 	if t.Name.Package == n.myPackage {
-		return "DeepCopy_" + pubName
+		return "DeepCopy" + pubName
 	}
-	return fmt.Sprintf("%s.DeepCopy_%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
+	return fmt.Sprintf("%s.DeepCopy%s", n.tracker.LocalNameOf(t.Name.Package), pubName)
```

Comment:
If we don't fix all lint issues then people who generate code need to go out of their way to tell lint to skip these files - which is really annoying. Any reason why we don't want to follow lint's guidance?